src/test/java/seatsio/events/UpdateEventTest.java

```diff
@@ -147,4 +149,23 @@ public void updateDate() {
         assertThat(retrievedEvent.date).isEqualTo(LocalDate.of(2022, 1, 6));
     }
 
+    @Test
+    public void updateIsInThePast() {
+        String chartKey = createTestChart();
+        Season season = client.seasons.create(chartKey, new SeasonParams().eventKeys(List.of("event1")));
+        Event event = client.events.retrieve("event1");
+        assertThat(event.isInThePast).isFalse();
+
+        client.events.update("event1", new UpdateEventParams().withIsInThePast(true));
+
+        Event retrievedEvent = client.events.retrieve(event.key);
+        assertThat(retrievedEvent.isInThePast).isTrue();
+
+        try {
```

Comment:
I wouldn't test events in the past being immutable - that's covered by the tests in `seatsio-core-v2`.